CelesteNet.Shared/DataTypes/DataLowLevelUDPInfo.cs

```diff
@@ -0,0 +1,96 @@
+namespace Celeste.Mod.CelesteNet.DataTypes {
+    /*
+    While debugging the old UDP protocol, I got so fed up with it that I created
```

Comment:
While this is `git blame`able, I'd still prefer you signed this lengthy comment off.